beets/dbcore/db.py

```diff
@@ -342,15 +342,17 @@ def __delattr__(self, key):
 
     # Database interaction (CRUD methods).
 
-    def store(self):
+    def store(self, fields_to_store=None):
         """Save the object's metadata into the library database.
         """
```

Comment:
This is just cosmetic, but I would prefer to use (a) a simpler parameter name like `fields` and (b) documentation in the docstring here to explain exactly what it does.